Remake get started (#1091)
Normation/rudder-doc: src/get-started/modules/ROOT/pages/node-management/node.adoc

* The `server` we already used
* A `node` machine

> As you used Vagrant to create the platform, it has provisoned the node in:

Reviewer: ```suggestion
If you used Vagrant to create the platform, it has provisoned the node by:
```
